src/documents/presentations/web-components-london-soledad-penades.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "An Introduction to Web Components"
+authors: [soledad_penades]
+event: Web Components London
+date: 2015-01-13
+image: web-components-london-soledad-penades.png
+category: presentations
+layout: single
+---
+
+Soledad Penades, Mozilla Web Apps Engineer, was the opening speaker for the first ever London Web Components meetup!
```

Comment:
Speaker introduction is not needed since it can be found in the Author card next to the post.